### PR TITLE
CFD-335 : linked groups should not appear in the discussion creation form.

### PR DESCRIPTION
**Pull request by YuriGizra**
#637

### CODE REVIEW

**Comment by solker** [comment]
Behat test failed on trevis but it's good on my local machine. 
Also version of danialfarid-angular-file-upload plugin changed from latest to 1.4.0 because on new version they rename files.

File: capacity4more/behat/features/group_access.feature [hunk 1]
```diff
@@ -66,3 +66,33 @@ Feature: Group access
     When a group "New group" with "Public" access is created with group manager "isaacnewton"
     Then I should be on the homepage
     And I should see "The group you requested is pending review by one of the administrators."
+
+  @api
+  Scenario: Check redirect to homepage  if no values are provided via URL.
```
Comment (solker) [inline capacity4more/behat/features/group_access.feature]: Change to scenario outline

**Comment by HelenaEksler** [comment]
@solker Re-roll, please.

**Comment by solker** [comment]
Re-rolled, waiting for finish tests.

**Comment by solker** [comment]
@HelenaEksler On my localhost test finished successful.

File: capacity4more/themes/c4m/kapablo/template.php [hunk 2]
```diff
@@ -10,3 +10,32 @@
 include_once 'theme/pager.inc';
 include_once 'theme/preprocess.inc';
 include_once 'theme/search.inc';
+
+/**
+ * Implements hook_preprocess_HOOK().
```
Comment (HelenaEksler) [inline capacity4more/themes/c4m/kapablo/template.php]: Fix docs.

**Comment by HelenaEksler** [comment]
@solker After group creation message is not appearing. Seems hook_status_message_preprocess is causing this. Check, please.

**Comment by HelenaEksler** [comment]
@zero2one @MPParsley PR is ready for review.

**Comment by nader77** [comment]
@HelenaEksler Needs a re-roll.

**Comment by zero2one** [comment]
@HelenaEksler  needs a re-roll to fix the broken build

**Comment by solker** [comment]
@zero2one re-rolled. Waiting Travis.

**Comment by HelenaEksler** [comment]
Closed in favor of #715.

**Pull request closed without merging**